galaxy_ng/app/settings.py

```diff
@@ -1,5 +1,17 @@
+import os
+
+
+PROMETHEUS_BEFORE_MIDDLEWARE = 'django_prometheus.middleware.PrometheusBeforeMiddleware'
+PROMETHEUS_AFTER_MIDDLEWARE = 'django_prometheus.middleware.PrometheusAfterMiddleware'
+PROM_ACCT_BEFORE = 'galaxy_ng.contrib.metrics_middleware.AccountEnhancedMetricsBeforeMiddleware'
+PROM_ACCT_AFTER = 'galaxy_ng.contrib.metrics_middleware.AccountEnhancedMetricsAfterMiddleware'
+ACCOUNT_ENHANCED_METRICS = os.environ.get('ACCOUNT_ENHANCED_METRICS', False) in (True, 'True')
```

Comment:
We use `1 \ 0` for booleans outside python and `true \ false` because of dynaconf. Please use one of these styles for consistency (or both)

For example:

```suggestion
ACCOUNT_ENHANCED_METRICS = os.environ.get('ACCOUNT_ENHANCED_METRICS') in ('1', 'true')
```